pandas/tests/io/formats/style/test_to_latex.py

```diff
@@ -242,6 +242,25 @@ def test_multiindex_row(df):
     assert expected == s.to_latex(sparse_index=False)
 
 
+def test_multirow_naive(df):
+    ridx = MultiIndex.from_tuples([("A", "a"), ("A", "b"), ("B", "c")])
```

Comment:
I suggest that the letters ``A`` and ``B`` should not be repeated in both the index and columns.
What if you use these for the index?

``` python
ridx = MultiIndex.from_tuples([("X", "x"), ("X", "y"), ("Z", "z")])
```
I guess it will be easier to differentiate from the columns.

Reply:
fully agree. silly oversight.